### PR TITLE
Button functionality and styling

### PR DESCRIPTION
**Pull request by tbtommyb**
Refactor button component to use single event handler function.

### CODE REVIEW

**Comment by des-des** [comment]
@tbtommyb / @Jwhiles  merge master then re-assign me! Also, pull lacking description

**Comment by stevehopkinson** [comment]
@des-des Master merged and reassigned.

File: src/components/App.js [hunk 1]
```diff
@@ -8,12 +8,26 @@ import '../styles/App.css';
...
+    obj[label]()
+  };
+
   render() {
     return (
       <div className='container'>
-        <Menu />
-        <Message/>
+        <Menu handleEvent={ this.handleEvent } />
+        {this.state.message ? <Message/> : null}
```
Comment (des-des) [inline src/components/App.js]: fyi, I think `this.state.message && <Message/>` works

File: src/components/Message.js [hunk 2]
```diff
@@ -18,9 +18,10 @@ class Message extends React.Component {
     return (
       <div className="message">
         <p>{conversation.text}</p>
-        {conversation.responses.map(response =>
+        {conversation.responses.map((response, i) =>
           <ResponseOption updateMessage={this.updateMessage}
-                          response={response} />
+                          response={response}
+                          key={i} />
```
Comment (des-des) [inline src/components/Message.js:24]: :tada:

File: src/styles/index.css [hunk 3]
```diff
@@ -55,9 +55,32 @@ body {
 }
 
 
+
 .button {
   width: 25%;
-  height: 30px;
+  box-sizing: border-box;
+	box-shadow:inset 0px 1px 0px 0px #fff6af;
```
Comment (des-des) [inline src/styles/index.css]: indentation

File: src/styles/index.css [hunk 4]
```diff
@@ -55,9 +55,32 @@ body {
...
+	cursor:pointer;
+	color:#333333;
+	font-size:15px;
+	font-weight:bold;
+	padding:6px 24px;
+	text-decoration:none;
+	text-shadow:0px 1px 0px #ffee66;
+}
+.button:hover {
+	background:linear-gradient(to bottom, #ffab23 5%, #ffec64 100%);
```
Comment (des-des) [inline src/styles/index.css]: indentaion

File: src/styles/index.css [hunk 5]
```diff
@@ -55,9 +55,32 @@ body {
...
+	filter:progid:DXImageTransform.Microsoft.gradient(startColorstr='#ffab23', endColorstr='#ffec64',GradientType=0);
+	background-color:#ffab23;
+}
+.button:active {
+	position:relative;
```
Comment (des-des) [inline src/styles/index.css]: 2 or 4 spaces?

File: src/components/App.js [hunk 6]
```diff
@@ -8,12 +8,26 @@ import '../styles/App.css';
 class App extends Component {
   constructor(props) {
     super(props);
+    this.state = { message: false }
+    this.handleEvent = this.handleEvent.bind(this);
   }
+
+  handleEvent (label) {
+    const obj = {
```
Comment (des-des) [inline src/components/App.js]: obj is bad name

**Review by des-des — CHANGES_REQUESTED** [review]

**Review by des-des — APPROVED** [review]

**Comment by des-des** [comment]
Happy to merge as soon as you assign me!

**Pull request merged**